### ISSUE TITLE
Conditions are not always saved properly for timeslots

### ISSUE THREAD

**Issue by andrey-yantsen** [nielsfaber/scheduler-card#415]
Unfortunately, I'm not sure about the reproduce case, but somehow I have some timeslots without conditions:

```bash
$ cat scheduler.storage | jq '.data.schedules[] | {name: .name, min_conditions: (([.timeslots[].conditions | length]) | min), max_conditions: (([.timeslots[].conditions | length]) | max)} | select(.min_conditions != .max_conditions)'
{
  "name": "Study climate weekdays @ Andrey at home",
  "min_conditions": 0,
  "max_conditions": 1
}
{
  "name": "Study climate weekends @ someone home",
  "min_conditions": 0,
  "max_conditions": 1
}
{
  "name": "Bedroom and study climate @ nobody home",
  "min_conditions": 0,
  "max_conditions": 1
}
{
  "name": "Corridor and lounge climate @ nobody home",
  "min_conditions": 0,
  "max_conditions": 1
}
```

I'll remove the conditions from those schedules and apply them again, and try to track when a condition could disappear from the config.

P.S. I added those schedules after the fix for #351.

**Comment by andrey-yantsen** [comment]
Found it! When you add a new timeslot to already configured schedule — the new one is saved without the conditions.

**Comment by nielsfaber** [comment]
Thanks for reporting this.
I thought I fixed this issue, apparently not.
I will check it out when I find the time for it.

**Comment by nielsfaber** [comment]
@andrey-yantsen  Could you confirm this is solved after upgrading to the 2.1.4 version?
Thanks!

**Comment by andrey-yantsen** [comment]
@nielsfaber yup, working like a charm, thanks!
